src/System.Private.CoreLib/shared/System/FormattableString.cs

```diff
@@ -73,6 +73,29 @@ public static string Invariant(FormattableString formattable)
             return formattable.ToString(Globalization.CultureInfo.InvariantCulture);
         }
 
+        /// <summary>
+        /// Format the given object in the current culture. This static method may be
+        /// imported in C# by
+        /// <code>
+        /// using static System.FormattableString;
+        /// </code>.
+        /// Within the scope
+        /// of that import directive an interpolated string may be formatted in the
+        /// current culture by writing, for example,
+        /// <code>
+        /// CurrentCulture($"{{ lat = {latitude}; lon = {longitude} }}")
+        /// </code>
+        /// </summary>
+        public static string CurrentCulture(FormattableString formattable)
+        {
+            if (formattable == null)
+            {
+                throw new ArgumentNullException(nameof(formattable));
+            }
+
+            return formattable.ToString(Globalization.CultureInfo.CurrentCulture);
```

Comment:
FWIW, this could just be:
```C#
return formattable.ToString();
```
since `ToString` is implemented as `return ToString(Globalization.CultureInfo.CurrentCulture);` below?

But what you have is fine, too.